R/dash.R

```diff
@@ -573,10 +583,11 @@ Dash <- R6::R6Class(
           </body>
         </html>',
         private$name,
-        render_dependencies(depsCSS, local = private$serve_locally),
+        render_dependencies(depsCSS, local = private$serve_locally, prefix=self$config$routes_pathname_prefix),
         to_JSON(self$config),
-        render_dependencies(depsScripts, local = private$serve_locally)
+        render_dependencies(depsScripts, local = private$serve_locally, prefix=self$config$routes_pathname_prefix)
```

Comment:
similarly here

Reply:
fixed in cc3ea048f7c2594e5efd137afcfb76324adba34d